app/models/vote.rb

```diff
@@ -0,0 +1,17 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :pizza
+
+  validates :pizza_id, presence: true, numericality: true
+  validates :user_id, presence: true, numericality: true
```

Comment:
The database relationship will enforce the fact that these fields are required.  

I would suggest a validation for the uniqueness of pizza and user combination, so the same user can't vote twice for the same pizza.

```suggestion
  validates :pizza_id, uniqueness: { scope: :user_id }
```